### ISSUE TITLE
Error 500 if image path is wrong or person is missing

### ISSUE THREAD

**Issue by LiaraAlis** [paulmassen/grav-plugin-seo#70]
If an image is entered for Twitter or Facebook where the path is not correct, an error 500 occured on the werbsite. 
The same applies if the microdata is activated for persons, but no person is entered.

This should not happen. The template should check if the file exists and there should be error messages when saving a page.

**Comment by paulmassen** [comment]
You are right, and coincidentally I ran into the same problem this morning.

It will be fixer on next update. Thanks

**Comment by paulmassen** [comment]
Fixed [here](https://github.com/paulmassen/grav-plugin-seo/commit/8225214180ef75fe96a5417077a45b5763db45f9)
7 years later...sorry

Thanks
